Doc changes for stack management and grouped feature privileges (#80486)
elastic/kibana: docs/management/advanced-options.asciidoc

. Enter a new value for the setting.
. Click *Save changes*.

[float]
=== Required permissions

The `Advanced Settings` {kib} privilege is required to access *Advanced Settings*.

> You can add this privilege in *Stack Management > Security > Roles*.

Reviewer: ```suggestion
To add the privilege, open the menu, then click *Stack Management > Roles*.
```
